src/controllers/auth/logout.ts

```diff
@@ -0,0 +1,9 @@
+import * as Interfaces from "../../interfaces/index";
+import * as Utils from "../../utils/index";
+
+const logout: Interfaces.Middlewares.Sync = (req, res) => {
+  req.logOut(() => false);
+  return res.json(Utils.Response.success("Logged Out Successfully"));
```

Comment:
You should only send the logout response when actually logged out, that is inside the callback.

Change it to

```js
req.logout((err)=>{
    if(err) // do something when error in logout
    else return res.json(Utils.Response.success("Logged Out Successfully"));
})
```